algorithms/dp/bitmasking_tsp.py

```diff
@@ -1,15 +1,187 @@
+import math
+
+MAX_ROW = 12
+MAX_COLUMN = 12
+MAX_HOUSE = 12
+
+# neighbor nodes position
+D_X = [-1, 0, 0, 1]
+D_Y = [0, 1, -1, 0]
+
+"""
+Check that the given nodes is valid
+
+:param nodes: The nodes to check
+:param nbRow: The number of rows
+:param nbColumn: The number of columns
+:returns: Nothing but raises error if invalid nodes
+"""
+def check_argument(nodes: list, nbRow: int, nbColumn: int):
```

Comment:
I think it would be more logical to place _check_argument_ right above _initialization_.

Reply:
I agree with this change